ERP-BACKEND/app/routers/crm.py

```
@@ -364,6 +404,231 @@ def delete_deal(deal_id: int, db: Session = Depends(get_db), current_user = Depe
+    return follow_up
+
+@router.get("/follow-ups", response_model=List[FollowUpResponse])
+def list_follow_ups(
+    skip: int = 0,
+    limit: int = 100,
+    entity_type: Optional[str] = None,
+    entity_id: Optional[int] = None,
+    completed: Optional[bool] = None,
+    priority: Optional[str] = None,
+    upcoming: Optional[bool] = None,
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)